gordo_components/workflow/workflow_generator/helpers.py

```diff
@@ -29,6 +39,6 @@ def patch_dict(original_dict: dict, patch_dictionary: dict) -> dict:
         `original_dict`
 
     """
-    diff = dictdiffer.diff(original_dict, patch_dictionary)
+    diff = dictdiffer.diff(original_dict, patch_dictionary, ignore=ignore)
```

Comment:
All these changes in this file are no longer required, correct?

Reply:
just a note...I'm ok with it.. but just exposing parameters to expose them is slightly strange IMO if this is left in.

Reply:
This isn't used any longer. I did some digging into the `dictdiffer` lib to find this, and while I'm not using it any longer, I thought that it is a useful thing to be aware of when doing this config patching. 

That said, I think the patching in general may be a confusing/problematic pattern to follow, so I'll just remove this with our discussion on changing things in the future.